src/ipc-handlers.ts

```diff
@@ -909,12 +909,24 @@ export async function showErrorMessageBox(
 		/Error invoking remote method '\w+': Error:/g,
 		''
 	);
-	await showMessageBox( event, {
+
+	console.error( filteredError );
+
+	const response = await showMessageBox( event, {
 		type: 'error',
 		message: title,
-		detail: error ? `${ message }\n\nError: ${ filteredError }` : message,
```

Comment:
Let's bring it back so the specific `error` can still be appended, but let's drop the 'Error: '.